doc/collapse.md

```diff
@@ -6,6 +6,17 @@ The **profile collapsing** function is a lightweight and flexible addition to th
 woltka tools collapse -i input.biom -m mapping.txt -o output.biom
 ```
 
+
+## Contents
+
+- [Use cases](#use-cases)
+- [Mapping file format](#mapping-file-format)
+- [Parameters](#parameters)
+- [Considerations](#considerations)
+
+
+## Use cases
+
 With this tool one can achieve the following goals:
 
 1. Translate feature IDs into names or descriptions.
```

Comment:
Should we add an extra point here (within 1): Translate genome IDs to functional proteins (get functional profiles) ?

Reply:
Perhaps worth adding also a reference here to the `Considerations` below; something like: Please read the `Considerations` below.

Reply:
> 
> 
> Should we add an extra point here (within 1): Translate genome IDs to functional proteins (get functional profiles) ?

I think I might be missing something here Antonio. To do the functional analysis, one would run the `classify` command and not `collapse`. One could collapse a genome id + orf id into a protein id, but a genome id  to protein would correspond to all the ORFs of that genome.

Reply:
Excellent question, I guess this means that the difference between collapse and classify is not clear and having more details will be useful. Perhaps having a simple page with this information + the comparison, my other comment will be useful. @qiyunzhu, what do you think?

Reply:
@antgonza thanks for the review!

1. The `comparison` section is further elaborated in `collapse.md` (The former has a link to the latter). Here is the elaboration: https://github.com/qiyunzhu/woltka/blob/upgrade/doc/collapse.md#considerations.
2. The suggestion "Translate genome IDs to functional proteins" may not apply here. This is because one cannot collapse genomes into functional proteins. (Technically one can but that will be no different from PICRUSt...)
3. Good suggestion! Let me add the reference.
4. I thought a bit, and currently I tend to refrain from making a separate page. My rationales are:
  - `collapse` is a simple utility whereas `classify` is the main module. So the comparison is only relevant when discussing `collapse`.
  - Several operations can affect the statistical property of the feature table, and one-to-many collapsing is but one of them. It may be worth to have a page dedicated to these questions.
  - Currently, `classify` cannot handle non-tree structures (especially directed acyclic graphs, which cover most functional classification systems), so we have to over-emphasize on `collapse` as a complement to `classify`. But implementing this new function is in our plan.

Reply:
Sorry, for the confusion. I meant "Translate per gene IDs to functional proteins", for example: from woltka genes to uniref proteins

Reply:
Makes sense to me! I have updated the PR.